lib/targets_fuzz.go

```diff
@@ -0,0 +1,36 @@
+// +build gofuzz
+
+package vegeta
+
+import (
+	"bytes"
+	"net/http"
+)
+
+// FuzzHTTPTargeter tests decoding an HTTP encoded target list.
+func FuzzHTTPTargeter(fuzz []byte) int {
+	targeter := NewHTTPTargeter(
+		bytes.NewReader(fuzz),
+		[]byte{},
+		http.Header{},
+	)
+	_, err := ReadAllTargets(targeter)
+	if err != nil {
+		return 0
+	}
+	return 1
+}
+
+// FuzzJSONTargeter tests decoding a JSON encoded target list.
+func FuzzJSONTargeter(fuzz []byte) int {
+	targeter := NewJSONTargeter(
+		bytes.NewReader(fuzz),
+		[]byte{},
```

Comment:
Any chance we can cover these default body and header arguments from the fuzz data?

Reply:
Added this defaults fuzzing. Also enhanced the header extraction everywhere to handle multivalue headers.